src/OpenLoco/GameCommands.cpp

```diff
@@ -120,97 +34,108 @@ namespace OpenLoco::GameCommands
-        nullptr,               // 47                                 0x0045436B
-        nullptr,               // 48                                 0x00455943
-        nullptr,               // 49                                 0x00496C22
-        nullptr,               // 50                                 0x0049711F
-        nullptr,               // 51                                 0x004A6FDC
-        nullptr,               // 52                                 0x004A734F
-        nullptr,               // 53                                 0x0047AF0B
-        nullptr,               // remove_industry = 54,              0x0042ECFC
-        nullptr,               // build_company_headquarters = 55,   0x0042EEAF
-        nullptr,               // 56                                 0x00492C41
-        nullptr,               // 57                                 0x00493559
-        nullptr,               // 58                                 0x004267BE
-        nullptr,               // vehicle_abort_pickup_air = 59,     0x00426B29
-        nullptr,               // 60                                 0x00493AA7
-        nullptr,               // 61                                 0x00494570
-        nullptr,               // 62                                 0x0042773C
-        nullptr,               // vehicle_abort_pickup_water = 63,   0x004279CC
-        nullptr,               // 63                                 0x0042F6DB
-        nullptr,               // 64                                 0x00435506
-        nullptr,               // change_company_face = 66,          0x00469CCB
-        nullptr,               // load_multiplayer_map = 67,         0x00444DA0
-        nullptr,               // 68                                 0x0046F8A5
-        nullptr,               // 69                                 0x004454BE
-        nullptr,               // 70                                 0x004456C8
-        nullptr,               // send_chat_message = 71,            0x0046F976
-        nullptr,               // multiplayer_save = 72,             0x004A0ACD
-        nullptr,               // update_owner_status = 73,          0x004383CA
-        nullptr,               // vehicle_speed_control = 74,        0x004BAB63
-        nullptr,               // vehicle_order_up = 75,             0x00470CD2
-        nullptr,               // vehicle_order_down = 76,           0x00470E06
-        nullptr,               // vehicle_apply_shunt_cheat = 77,    0x004BAC53
-        nullptr,               // apply_free_cash_cheat = 78,        0x00438A08
-        nullptr,               // rename_industry = 79,              0x00455029
-        Vehicles::cloneVehicle // vehicle_clone = 80,                *NEW*
+
+    struct GameCommandInfo
+    {
+        GameCommand id;
+        GameCommandFunc implementation;
+        uintptr_t originalAddress; // original array: 0x004F9548
+        bool resumesGame;          // original array: 0x004F9688
```

Comment:
I find this variable name a bit misleading, wouldn't this mean that running game commands halts the state and resumes it if this true.

Reply:
what would you rather it be. I think its fine if you really want you could add a bit of extra info stating that if true will resume play of a paused game but imho I think its clear.

Reply:
Maybe I'm not sure exactly what this field does. If this is a table that states which actions can be executed while the game is paused then I would just say 'allowWhilePaused'. If the game command checks this field and forces the game to resume if its true then the name 'resumesGame' is a fitting name.

Reply:
There was a discussion about the name of this field earlier in the PR. The meaning is indeed that, if true, the game is resumed to execute this action. (If the game is already not paused, there is no difference.)

Reply:
`unpausesGame`?

Reply:
I'd be fine with that as well. @duncanspumpkin @ZehMatt ?

Reply:
I really do not care either way

Reply:
I've gone ahead and made the change.